tst/EnergyPlus/unit/PurchasedAirManager.unit.cc

```diff
@@ -499,6 +507,15 @@ TEST_F(ZoneIdealLoadsTest, IdealLoads_ExhaustNodeTest)
         "  ,                               !- Induced Air Outlet Node or NodeList Name",
         "  Zone Exhaust Node;              !- Inlet 1 Node Name",
 
+        "  Schedule:Constant,",
+        "    DXHeatingCoilFuelEffSched,    !- Name",
+        "    AnyValue,                     !- Schedule Type Limits Name",
+        "    2.0;                          !- Field 1",
+
+        "  Schedule:Constant,",
+        "    DXCoolingCoilFuelEffSched,    !- Name",
+        "    AnyValue,                     !- Schedule Type Limits Name",
+        "    3.0;                          !- Field 1",
```

Comment:
Two constant schedule objects were added to an existing unit test for testing the fuel efficiency inputs test.